_ide_helper.php

```
@@ -4008,6 +4036,20 @@
+        /**
+         * Assert if a job was pushed exactly once.
+         *
+         * @param string|\Closure $command
+         * @param int $times
+         * @return void
+         * @static
+         */
+        public static function assertDispatchedOnce($command)
+        {
+            /** @var \Illuminate\Support\Testing\Fakes\BusFake $instance */
+            $instance->assertDispatchedOnce($command);
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$instance' which will lead to PHP notices. (Clean Code Rules)

(UndefinedVariable)